src/status_im/contexts/wallet/wallet_connect/core.cljs

```diff
@@ -116,3 +116,19 @@
    :url          (get-in session [:peer :metadata :url])
    :accounts     (get-in session [:namespaces :eip155 :accounts])
    :disconnected false})
+
+(defn filter-operable-accounts
+  [accounts]
+  (filter #(and (:operable? %)
+                (not (:watch-only? %)))
+          accounts))
+
+(defn filter-sessions-for-account-addresses
+  [account-addresses sessions]
+  (filter (fn [{:keys [accounts]}]
+            (some (fn [account]
+                    (some (fn [account-address]
+                            (clojure.string/includes? account account-address))
+                          account-addresses))
+                  accounts))
+          sessions))
```

Comment:
are these helpers available elsewhere as well? Like in a sub or smth. Could be extracted into a helper maybe

Reply:
It's more like a function designed to help with processing sessions we've just got from SDK, that's why it's not a sub.
But isn't it already like a helper? I moved it to here because it can potentially be useful in other places as well.

Reply:
ah no, I though you copied it from a sub or smth and this logic exists elsewhere as well. If that's not the case, then it's all good!